emacs/init.sls

```diff
@@ -1,2 +1,108 @@
-emacs:
-  pkg.installed
+{% from 'emacs/settings.sls' import emacs with context %}
+
+{%- if emacs.from_pkg == True %}
+## This is the default behavior, milk-toast if you will.
+boring-emacs:
+  pkg.installed:
+    - name: emacs
+
+{% else %}
+# You have chosen wisely but your journey to the great Basillica of St. Stallman has only begin!
+{% set build_dir = '/usr/local/src' %}
```

Comment:
Sorry my comment was specifically related to line 11 regarding setting the `build_dir` value.